### DISCUSSION TITLE
Integration to NextCloud just like {Google|One}drive|DropBox ?

### DISCUSSION THREAD

**Discussion by hvisage** [jgraph/drawio#2734] [Ideas]
Having made the decisions to move off the big-tech to mostly self hosted software by my client, {Google|One}drive|DropBox had been dropped in favor of NextCloud.

Even though the local filesystem is an option, whenever I'm roaming, the online diagrams.net is just what I need to share with their roaming users... just.... I can only use the local disk instead of direct to NextCloud.

**Comment by davidjgraph** [comment]
https://apps.nextcloud.com/apps/drawio
https://github.com/pawelrojek/nextcloud-drawio
